dts/arm/nucleo_f401re.dts

```diff
@@ -27,3 +27,31 @@
 	current-speed = <115200>;
 	status = "ok";
 };
+
+&flash0 {
+	partitions {
```

Comment:
Can you add this comment:

+                               /*
+                                * If chosen's zephyr,code-partition
+                                * is unset, the image will be linked
+                                * into the entire flash device.  If
+                                * it points to an individual
+                                * partition, the code will be linked
+                                * to, and restricted to that
+                                * partition.
+                                */